### DISCUSSION TITLE
AttributeError: module 'brevitas.onnx' has no attribute 'export_finn_onnx'

### DISCUSSION THREAD

**Discussion by bleachman08** [Xilinx/finn#951] [Q&A]
I am following the end to end example of cybersecurity, I am in the 1st notebook.
I have executed the training and testing process successfully but while exporting to FINN-ONNX, I am getting the following error
AttributeError: module 'brevitas.onnx' has no attribute 'export_finn_onnx'

**Comment by fpjentzsch** [comment]
Hi, which branch of FINN are you using? Could you try with the dev branch?

**Comment by iordanouki** [comment]
Hi,

I have the same issue.

**Reply by dianamahdi** [reply]
Hi , if you are working on the dev branch , replace 'export_finn_onnx' with 'export_qonnx'

**Reply by iordanouki** [reply]
Thanks!

**Reply by bleachman08** [reply]
Hi, i am currently not able to pass the quicktest of FINN, could you please help and let me know how you escaped all the errors?

**Reply by sainadh812** [reply]
import brevitas.onnx as bo
from brevitas.quant_tensor import QuantTensor


# Export to ONNX
bo.export_qonnx(
    model_for_export, export_path=ready_model_filename, input_t=input_qt)


i am getting error ,what i should do ?

**Reply by sainadh812** [reply]
---> 20 bo.export_finn_onnx(
     21     model_for_export, export_path=ready_model_filename, input_t=input_qt
     22 )

AttributeError: module 'brevitas.onnx' has no attribute 'export_finn_onnx'

**Comment by sainadh812** [comment]
!pip install --upgrade brevitas
import brevitas
print(brevitas.__version__)

import brevitas.onnx as bo
dir(bo)


0.10.0
['__builtins__',
 '__cached__',
 '__doc__',
 '__file__',
 '__loader__',
 '__name__',
 '__package__',
 '__path__',
 '__spec__',
 'enable_debug',
 'export_brevitas_onnx',
 'export_onnx_qcdq',
 'export_onnx_qop',
 'export_qonnx']
